### DISCUSSION TITLE
Merge Order

### DISCUSSION THREAD

**Discussion by jgoll7** [jgoll7/Tafe22S211#4] [General]
Hi, just checking in about the merging process.
From my understanding we are merging the mortgage feature into the development branch, then the currency feature into development.
Then the development branch into the release branch where we will do a final test before merging into the main branch.
Is that correct?

**Comment by MoonlitBeans** [comment]
Yup. We can merge in any order, but you can do the merge first if you like. I need to do some additional changes.

**Reply by jgoll7** [reply]
Ok great, going to merge my feature to the development branch now.
